src/app/common/subscriptions/SubscriptionDetails.vue

```diff
@@ -1,97 +1,69 @@
 <template>
-  <div>
-    <div v-if="details.globalInstanceId || details.controlPlaneInstanceId || details.connectTime || details.disconnectTime">
-      <h5 class="overview-tertiary-title">
-        General Information:
-      </h5>
-
-      <DefinitionList>
-        <DefinitionListItem
-          v-if="details.globalInstanceId"
-          :term="t('http.api.property.globalInstanceId')"
-        >
-          {{ details.globalInstanceId }}
-        </DefinitionListItem>
-
-        <DefinitionListItem
-          v-if="details.controlPlaneInstanceId"
-          :term="t('http.api.property.controlPlaneInstanceId')"
-        >
-          {{ details.controlPlaneInstanceId }}
-        </DefinitionListItem>
-
-        <DefinitionListItem
-          v-if="details.connectTime"
-          :term="t('http.api.property.connectTime')"
-        >
-          {{ formatIsoDate(details.connectTime) }}
-        </DefinitionListItem>
-
-        <DefinitionListItem
-          v-if="details.disconnectTime"
-          :term="t('http.api.property.disconnectTime')"
-        >
-          {{ formatIsoDate(details.disconnectTime) }}
-        </DefinitionListItem>
-      </DefinitionList>
-    </div>
-
-    <div
-      v-if="detailsIterator"
-      class="columns mt-4"
-      style="--columns: 4"
-    >
-      <template
-        v-for="(item, label) in detailsIterator"
-        :key="label"
-      >
-        <div v-if="Object.keys(item).length > 0">
-          <h6 class="overview-tertiary-title">
-            {{ t(`http.api.property.${label}`) }}:
-          </h6>
-
-          <DefinitionList>
-            <DefinitionListItem
-              v-for="(value, property) in item"
-              :key="property"
-              :term="t(`http.api.property.${property}`)"
-            >
-              {{ formatError(formatValue(value)) }}
-            </DefinitionListItem>
-          </DefinitionList>
-        </div>
-      </template>
-    </div>
-
+  <div class="stack">
     <KAlert
-      v-else
+      v-if="statuses.length === 0"
       appearance="info"
-      class="mt-4"
     >
       <template #alertIcon>
         <KIcon icon="portal" />
       </template>
 
       <template #alertMessage>
-        There are no subscription statistics for <strong>{{ details.id }}</strong>
+        {{ t('common.detail.subscriptions.no_stats', { id: props.subscription.id }) }}
```

Comment:
Nit: I'm not sure that this should go in `common`. I'm guessing this should be on the i18n for the component (ideally the route component but I don't think we are there yet). If we are not careful `common` is just going to be a massive bucket full of all sorts.

I think the fact that we further namespace this under `.subscriptions` means its not "common"

Reply:
But it’s also not specific to this module as both zone and DPP views use this component.

Reply:
Right so if should go on the component i18n keys

i.e. `components.subscription_details.stuff`. 

The larger question about both zone and DPP views using this component, to me it would make sense that this component should live in the dataplane module and reused from zones, but lets leave this for now, we can circle back later.

Feel free to resolve.